docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ "$1" = 'bety' ]; then
+    # Wait for Postgres to start
+    # http://www.onegeek.com.au/articles/waiting-for-dependencies-in-docker-compose
+    WAIT=0
+    while ! nc -z $PG_PORT_5432_TCP_ADDR $PG_PORT_5432_TCP_PORT; do
+      sleep 1
+      WAIT=$(($WAIT + 1))
+      if [ "$WAIT" -gt 15 ]; then
+        echo "Error: Timeout wating for Postgres to start"
+        exit 1
+      fi
+    done
+
+    # Move database config file into config directory & add host/port
+    cd /home/bety/config
+    /bin/cp /home/bety/docker/database.yml database.yml
+    /bin/sed -i "/host:/ s/$/ $PG_PORT_5432_TCP_ADDR/" database.yml
+    /bin/sed -i "/port:/ s/$/ $PG_PORT_5432_TCP_PORT/" database.yml
```

Comment:
sed -i will create a backup and replace the file you are working on. If not given an argument it will just replace the original file. On a mac this does not work however and you will need to specify an argument for -i. The PG_PORT are indeed environment variables automatically set by docker.